docs/guide/dev/deep_dive/errors/README.md

```diff
@@ -61,6 +61,14 @@ This article contains the list of CKEditor error codes and their explanation. Th
 	* `plugin`: The name of the plugin that cannot be removed.
 	* `requiredBy`: The name of the plugin whose requirements block the removal.
 
+## editor-plugin-conflict
+
+* Location: `core/editor.js`
```

Comment:
I'm not sure if it's correct location for this error. In fact it's raised in particular plugins. After changes requested in https://github.com/ckeditor/ckeditor-dev/pull/2246#pullrequestreview-139034733 it will be raised from `core/tools.js`.